Assets/MixedRealityToolkit.Providers/UnityAR/Editor/UnityARCameraSettingsProfileInspector.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Editor;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using System.Linq;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.CameraSystem
+{
+    [CustomEditor(typeof(UnityARCameraSettingsProfile))]
+    public class UnityARCameraSettingsProfileInspector : BaseMixedRealityToolkitConfigurationProfileInspector
+    {
+        private const string ProfileTitle = "Unity AR Foundation Camera Settings";
+        private const string ProfileDescription = "";
```

Comment:
Should this be filled in?

Reply:
it is optional. not all profiles have a description